add-domain

```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
+mail-e9eredelivery.com
```

Comment:
drill -T mail-e9eredelivery.com
.       518400  IN      NS      m.root-servers.net.
.       518400  IN      NS      f.root-servers.net.
.       518400  IN      NS      e.root-servers.net.
.       518400  IN      NS      b.root-servers.net.
.       518400  IN      NS      k.root-servers.net.
.       518400  IN      NS      c.root-servers.net.
.       518400  IN      NS      g.root-servers.net.
.       518400  IN      NS      h.root-servers.net.
.       518400  IN      NS      d.root-servers.net.
.       518400  IN      NS      j.root-servers.net.
.       518400  IN      NS      i.root-servers.net.
.       518400  IN      NS      a.root-servers.net.
.       518400  IN      NS      l.root-servers.net.
com.    172800  IN      NS      k.gtld-servers.net.
com.    172800  IN      NS      g.gtld-servers.net.
com.    172800  IN      NS      e.gtld-servers.net.
com.    172800  IN      NS      h.gtld-servers.net.
com.    172800  IN      NS      l.gtld-servers.net.
com.    172800  IN      NS      i.gtld-servers.net.
com.    172800  IN      NS      d.gtld-servers.net.
com.    172800  IN      NS      a.gtld-servers.net.
com.    172800  IN      NS      m.gtld-servers.net.
com.    172800  IN      NS      j.gtld-servers.net.
com.    172800  IN      NS      c.gtld-servers.net.
com.    172800  IN      NS      b.gtld-servers.net.
com.    172800  IN      NS      f.gtld-servers.net.
com.    900     IN      SOA     a.gtld-servers.net. nstld.verisign-grs.com. 1626793872 1800 900 604800 86400